src/swoop/api/app.py

```diff
@@ -20,6 +21,7 @@ def get_app() -> FastAPI:
     async def startup_event():
         """Connect to database on startup."""
         await connect_to_db(app)
+        init_workflows_config(app)
```

Comment:
I think these would be best flipped, in case workflow config loading fails (missing file, bad yaml...) we aren't opening up connections to the database that will immediately get torn back down (and in what is likely a crash loop, such that it could create significant load on the database server).

Note that in the coming ticket to use the payload caching when processing a workflow, another element of loading the workflow configs will be to parse the cache key include/exclude lists, so not only does the yaml have to be the right format, but those will need to be as well.

```suggestion
        init_workflows_config(app)
        await connect_to_db(app)
```